src/index.js

```diff
@@ -1,11 +1,11 @@
-import StripePlugin from './stripe';
+// import StripePlugin from './stripe';
```

Comment:
@gonzalojpv can you please remove commented out code?

Reply:
@semanticmx sure!, the fix in last commit